src/components/EventDateFilter/EventDateFilter.sass

```diff
@@ -55,6 +55,7 @@
     position: relative
     cursor: pointer
     font-size: 20px
+    border: 1px solid transparent
```

Comment:
Can you add a comment for this?  To a delete-happy passerby like myself, I'd probably axe this thinking it wasn't serving a purpose